contrib/builder/rpm/amd64/generate.sh

```diff
@@ -106,8 +106,12 @@ for version in "${versions[@]}"; do
 
 	case "$from" in
 		opensuse:*)
-			packages=( "${packages[@]/btrfs-progs-devel/libbtrfs-devel}" )
+			packages=( "${packages[@]/btrfs-progs-devel/libbtrfs-devel }" )
 			packages=( "${packages[@]/pkgconfig/pkg-config}" )
+			if [[ "$from" == "opensuse:13."* ]]; then
+				packages+=systemd-rpm-macros
```

Comment:
ya how is that working

On Wed, Feb 3, 2016 at 10:30 AM, Tianon Gravi <notifications@github.com>
wrote:

> In contrib/builder/rpm/amd64/generate.sh
> <https://github.com/docker/docker/pull/19972#discussion_r51764526>:
>
> >  			packages=( "${packages[@]/pkgconfig/pkg-config}" )
> > +			if [[ "$from" == "opensuse:13."* ]]; then
> > +				packages+=systemd-rpm-macros
>
> This should be +=( ... ) right? (so it stays an array)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/19972/files#r51764526>.
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>